php/README.md

```diff
@@ -0,0 +1,34 @@
+# HubSpot-php sample OAuth 2.0 app
```

Comment:
Since the repo contains [top-level](https://github.com/HubSpot/sample-apps-oauth/blob/main/README.md) README, please leave only setup instructions here, similar to how it's done in [node](https://github.com/HubSpot/sample-apps-oauth/blob/main/node/README.md).

Reply:
I've just deleted readme
There is only code base

Reply:
Why do you completely removed it? need to leave php instructions, no?